content/ch-quantum-hardware/Jaynes-Cummings-model.ipynb

```diff
@@ -66,36 +66,8 @@
     "from qiskit.compiler import transpile, assemble\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    qc_meas_spect.add_calibration(meas_spect_gate, (0,), meas_spect_sched, [freq, amp])
</pre><p>This should use variable <code>qubit</code> number instead of hard-coded <code>(0,)</code>.  Since you provide <code>meas_spect_gate</code> knowing parameters in the first argument, you don't need to explicitly give <code>[freq, amp]</code> here.  In addition, you need to transpile this circuit for qubit layout. </p><pre class="ql-syntax" spellcheck="false">physical_qc = transpile(qc_meas_spect, initial_layout=[qubit], coupling_map=backend.configuration().coupling_map)
</pre><p>Without this lowering, you cannot use qubit other than 0.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/qiskit-community/qiskit-textbook/pull/1541/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
Thank you for your comment.
OK. I will change .